dataactvalidator/config/sqlrules/fabs44_detached_award_financial_assistance_3.sql

```diff
@@ -8,9 +10,10 @@ FROM detached_award_financial_assistance AS dafa
 WHERE submission_id = {0}
     AND COALESCE(legal_entity_zip5, '') <> ''
     AND COALESCE(legal_entity_congressional, '') <> ''
-    AND NOT EXISTS (
-        SELECT 1
-        FROM zips AS z
-        WHERE z.zip5 = dafa.legal_entity_zip5
-            AND z.congressional_district_no = dafa.legal_entity_congressional
-    );
+    AND dafa.legal_entity_congressional
+        NOT IN (SELECT sc.congressional_district_no
+    	        FROM state_congressional AS sc,
+    	 	        zips as z
+    	        WHERE z.zip5 = dafa.legal_entity_zip5
+    	            AND z.state_abbreviation = sc.state_code
+    	            AND sc.census_year IS NULL OR sc.census_year >= 2000);
```

Comment:
Same as comment above in terms of COALESCE

Reply:
Updating to coalesce to 2010 (since should be the default)